x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/service/notifications/success_notification.tsx

```diff
@@ -49,8 +49,8 @@ const SuccessToastContent: React.FC<{ migration: RuleMigrationStats }> = ({ migr
       <EuiFlexItem>
         <FormattedMessage
           id="xpack.securitySolution.siemMigrations.rulesService.polling.successText"
-          defaultMessage="SIEM rules migration #{number} has finished translating. Results have been added to a dedicated page."
-          values={{ number: migration.number }}
+          defaultMessage="SIEM rules migration {name} has finished translating. Results have been added to a dedicated page."
+          values={{ name: migration.name }}
```

Comment:
For legacy migrations, this template generates texts like:

`SIEM rules migration SIEM rules migration #3 has finished translating. Results have been added to a dedicated page.`

What if we do the following:
```suggestion
          defaultMessage='Migration "{name}" has finished translating. Results have been added to the translated rules page.'
          values={{ name: migration.name }}
```

So the resulting text will be:
`Migration "SIEM rules migration #3" has finished translating. Results have been added to the translated rules page.`

wdyt?

(having feedback from a designer would also help)